content/millennium/r4/individuals/patient.md

```diff
@@ -9,4 +9,215 @@ title: Patient | R4 API
+* The `birthdate` parameter may be provided once with the following prefixes: 'ge', 'le', 'gt', 'lt', 'eq'
+
+### Headers
+
+ <%= headers %>
+
+### Example
+
+#### Request
+
+    GET https://fhir-open.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Patient?_id=4342009
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:r4_patient_bundle) %>
+
+### Errors
+
+The common [errors] and [OperationOutcomes] may be returned. In addition, a `422 Unprocessable Entity` will be returned when too many (>1000) patients qualify for the search criteria.
+
+## Retrieve by id
+
+List an individual Patient by its id:
+
+    GET /Patient/:id
+
+_Implementation Notes_
+
+* Direct secure email will not be returned.
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Headers
+
+<%= headers %>
+
+### Example
+
+#### Request
+
+    GET https://fhir-open.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Patient/4342009
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:r4_patient_entry) %>
+
+### Patient Combines Example
+
+Cerner Millennium supports the ability to logically merge a patient record into another patient record when both records are describing the same patient. This is known
+as a "patient combine". If necessary, this merging can later be undone by performing a "patient uncombine". When the requested patient record has been combined into another
+record, an inactive Patient entry will be returned which has a link to the current Patient entry. Entries for combined patients will only be returned when retrieving
+the entries directly by id. They will not be returned when searching with other parameters.
+
+The ability to perform patient combine or uncombine operations is not available through the Cerner Ignite platform.
+
+#### Request
+
+    GET https://fhir-open.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Patient/4860007
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:r4_combined_patient_entry) %>
+
+### Errors
+
+The common [errors] and [OperationOutcomes] may be returned.
+
+## Create
+
+Create a new Patient.
+
+    POST /Patient
+
+_Implementation Notes_
+
+* Only the body fields mentioned below are supported. Unsupported fields will be ignored.
+* Modifier fields should not be provided, and will cause the transaction to fail.
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', Accept: 'application/json+fhir', 'Content-Type': 'application/json+fhir'} %>
+
+### Body Fields
+
+Notes:
+
+* Birth Sex may be recorded as an extension.
+
+<%= definition_table(:patient, :create, :r4) %>
+
+### Example
+
+#### Request
+
+    POST https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Patient
+
+#### Body
+
+#### Response
+
+<%= headers status: 201 %>
+<pre class="terminal">
+    Cache-Control: no-cache
```

Comment:
spaces inside a pre tag are rendered as spaces on the site

Reply:
Yes, and we've used 4 spaces while rendering them for DSTU2. Are we changing this for R4?

Reply:
yes

Reply:
Removed spacing: https://github.com/cerner/fhir.cerner.com/pull/241/commits/8b4a41befb353cad101bd7e61edcbdcce61e32d5#diff-fb8aa00e1932d86c4425f2f33f20c7d1L191